gaas-ui/test/components/create-graph/create-graph.test.tsx

```diff
@@ -53,7 +53,7 @@ describe("CreateGraph UI component", () => {
             expect(select.text()).toBe("Store Type​​");
         });
         it("should have icon button", () => {
-            const fileButton = wrapper.find("button").at(0).find("svg");
+            const fileButton = wrapper.find("button").at(1).find("svg");
```

Comment:
I am sure this is very innocent but wondering why this was changed?

Reply:
The first button is now the schema builder button, which is a generic button with text. The second one is now one that has an icon, which is why the index has changed.